src/main/java/com/habicus/core/configuration/ServiceConfiguration/ServiceConfig.java

```diff
@@ -33,4 +34,9 @@
   public GoalService goalService() {
     return new GoalService();
   }
+
+  @Bean
+  UserService userService() {
```

Comment:
GoalService has public access modifier--does UserService need this as well?

Reply:
Fixed 🏁